fda-model-test/integrationtests/DefaultDataComputeOutcomes.cs

```diff
@@ -208,25 +208,34 @@ public void WithoutAnalytical_ScenarioResults(double expectedCommercialMeanEAD,
             Scenario scenario2 = new Scenario(futureYear, impactAreaScenarioSimulations);
             ScenarioResults scenarioResults2 = scenario2.Compute(randomProvider, convergenceCriteria);
             AlternativeResults alternativeResults = new Alternative().AnnualizationCompute(randomProvider, .025, 50, 1, scenarioResults, scenarioResults2);
-            IHistogram eadHistogram = alternativeResults.GetBaseYearEADHistogram(impactAreaID1, commercialDamageCategory);
+            Empirical empiricalEADDistribution = alternativeResults.GetBaseYearEADDistribution(impactAreaID1, commercialDamageCategory);
 
             //Act
             double actualCommercialMeanEAD = scenarioResults.MeanExpectedAnnualConsequences(impactAreaID1, commercialDamageCategory);
             double actualResidentialMeanEAD = scenarioResults.MeanExpectedAnnualConsequences(impactAreaID1, residentialDamageCategory);
             double actualMeanAAEQ = alternativeResults.MeanAAEQDamage(impactAreaID1, commercialDamageCategory);
-            double actualCommercialMeanEADFromAnotherSource = eadHistogram.Mean;
+            double actualCommercialMeanEADFromAnotherSource = empiricalEADDistribution.Mean;
             double tolerance = 0.11;
             double strictTolerance = 0.01;
             double commercialMeanEADSourcesRelativeDifference = Math.Abs(actualCommercialMeanEAD - actualCommercialMeanEADFromAnotherSource) / actualCommercialMeanEADFromAnotherSource;
             double commercialEADRelativeDifference = Math.Abs(actualCommercialMeanEAD - expectedCommercialMeanEAD) / expectedCommercialMeanEAD;
             double residentialEADRelativeDifference = Math.Abs(actualResidentialMeanEAD - expectedResidentialMeanEAD) / expectedResidentialMeanEAD;
+
+
+            //TODO: //EAD is constant over POA soq AAEQ = EAD
```

Comment:
I'd like to make TODOs into issues on github just to be sure they're tracked. Easy to get lost in comments.